careers/game/environment.py

```diff
@@ -10,12 +10,12 @@ def __init__(self, package_name=None):
         """Initialize the running Environment by setting global environment variables
         
         """
+        script_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I removed the resources[] array because I could find them being used. Maybe you intended them to be used in the future?

Reply:
Yes I was, but this change is okay.